<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentForm</class>
 <widget class="QDialog" name="studentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_studentCard">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Student Card Information:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_studentID">
       <property name="text">
        <string>Student ID:  @</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_studentID</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_studentID"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_collegeEnrolled">
       <property name="text">
        <string>Enrolled College:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_colleges</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_colleges">
       <item>
        <property name="text">
         <string>- Please Select -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>U.A. Whitaker College of Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>College of Arts &amp; Sciences</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lutgert College of Business</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>College of Education</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marieb College of Health &amp; Human Services</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other - Please Specify</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_other">
       <property name="text">
        <string>Other College:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_otherCollege</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_otherCollege"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_gradYear">
       <property name="text">
        <string>Graduation Year:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_gradYear</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_gradYear">
       <item>
        <property name="text">
         <string>- Please Select -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2024</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2025</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2026</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2027</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2028</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createStudent">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancelStudent">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
